sql/scriptdev2/scriptdev2.sql

```diff
@@ -1268,6 +1268,21 @@ UPDATE creature_template SET ScriptName='npc_tesla_coil' WHERE entry=16218;
 UPDATE creature_template SET ScriptName='boss_sapphiron' WHERE entry=15989;
 UPDATE gameobject_template SET ScriptName='go_sapphiron_birth' WHERE entry=181356;
 UPDATE creature_template SET ScriptName='boss_kelthuzad' WHERE entry=15990;
+-- BossAI Additions
+UPDATE creature_template SET ScriptName="npc_living_poison" WHERE Entry=16027;
+UPDATE creature_template SET ScriptName="npc_stoneskin_gargoyle" WHERE Entry=16168;
+UPDATE `creature_template` SET SpellList=1602801 WHERE entry IN (16028, 29324);
```

Comment:
Goes into db.

Reply:
where tho?

Reply:
```
UPDATE `creature_template` SET SpellList=1602801 WHERE entry IN (16028, 29324);
```
Normal update file. Thats why we name all of them.